ceph/ceph_admin/add.py

```diff
@@ -39,7 +39,7 @@ def add(self: DaemonProtocol, config: Dict):
         pos_args = config["pos_args"]
         node = pos_args[0]
         host_id = get_node_by_id(self.cluster, node)
-        host = host_id.shortname
+        host = host_id.hostname
```

Comment:
Why this sudden change? `shortname` and `hostname` are different. It might affect all tests which are using this module.

Reply:
switching to hostname was there from long time. this module wasn't updated, since it didn't throw exception for PSI/IBM  instances, where shortname and hostname are same.
https://github.com/red-hat-storage/cephci/commit/098f32cb1b713460269d93183aebf6c70b8ef1c6